### DISCUSSION TITLE
Some reliable HRV and physio datasets with open access?

### DISCUSSION THREAD

**Discussion by Przemekeke** [neuropsychology/NeuroKit#894] [General]
Hi,

I know that maybe that is not much related here but maybe someone could help me.
It seems to me that the repositories with HRV and physio data are pretty limited. There is the most known SWELL dataset, some kaggle datasets and probably this is the biggest database: https://physionet.org/about/database/. But some datasets are very limited or useful metadata is removed, and the most useful which I found are from the beginning of the century.

Has anyone tried to approach that problem some other way?
I understand that there is some data which might be a very valuable asset for someone and there are some privacy constraints of course.

**Comment by DominiqueMakowski** [comment]
Yes physionet is probably the biggest repository of cardiac activity data, but indeed in general datasets are of varying quality / useability / format.

With @danibene and @Tam-Pham we will soon be starting to think of something to facilitate access to large datasets that can be used to study HRV. I'll let you know once we come up with something
